service/worker/deployment/fx.go

```diff
@@ -144,11 +133,11 @@ func (s *workerComponent) newDeploymentActivities(name namespace.Name, id namesp
 	}
 }
 
-// TODO Shivam - place holder for now but will initialize activity rate limits (if any) amongst other things
-func (s *workerComponent) newDeploymentSeriesActivities(name namespace.Name, id namespace.ID) *DeploymentSeriesActivities {
-	return &DeploymentSeriesActivities{
-		activityDeps: s.activityDeps,
-		namespace:    name,
-		namespaceID:  id,
-	}
-}
+// // TODO Shivam - place holder for now but will initialize activity rate limits (if any) amongst other things
```

Comment:
this had to be removed since the service was not starting. I think @dnr's work will probably change this.